### PR TITLE
Creates Tasks hook and remove routes

### PR DESCRIPTION
**Pull request by jzeiders**
Kills un-needed routes, can re-add those ass neeeded in new sdk form

### CODE REVIEW

File: api/tasks.js [hunk 1]
```diff
@@ -37,38 +33,26 @@ router.post("/tasks", async (req, res) => {
...
-    return res.status(200);
-  } catch (e) {
-    return res.status(500).json({ err: e.message });
-  }
+router.put("/:id", async (req, res) => {
+  const updatedObj = req.body;
+  const taskId = updatedObj.id;
+  delete updatedObj.id;
+  await models.Task.update({ ...updatedObj }, { where: { id: taskId } });
+  return res.status(200);
```
Comment (WilhelmWillie) [inline api/tasks.js]: unless i missed something, is there a reason we don't return the updated task?

Comment (jzeiders) [reply api/tasks.js]: rolling the get hooks for now, only ones used, going to port the create routes as needed

File: lib/api-sdk/taskHooks.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const resourceRoute = Routes.TasksList;
+  const { data: allTasks, error } = useSWR<Task[], any>(
+    resourceRoute,
+    fetcherToSVRHandler(getAllTasksFetch),
+    {
+      initialData: initialModels
+    }
+  );
+
+  const activeTasks: ActiveTask[] = allTasks.filter(isActiveTask);
```
Comment (WilhelmWillie) [inline lib/api-sdk/taskHooks.tsx]: lovely

File: pages/scan.tsx [hunk 3]
```diff
@@ -180,7 +188,7 @@ Scan.getInitialProps = async ctx => {
 
   return {
     profile,
-    tasks: tasks.tasks // TODO -- update this once we've standardized our server/client stuff
+    tasks: activeTasks // TODO -- update this once we've standardized our server/client stuff
```
Comment (WilhelmWillie) [inline pages/scan.tsx]: ty, feel free to remove that comment

**Pull request merged**